packages/go/index.ts

```diff
@@ -784,6 +805,25 @@ async function waitForPortFile_(opts: {
 export async function prepareCache({
   workPath,
 }: PrepareCacheOptions): Promise<Files> {
+  // When building the project for the first time, there won't be a cache and
+  // `createGo()` will have downloaded Go to the global cache directory, then
+  // symlinked it to the local `cacheDir`.
+  //
+  // If we detect the `cacheDir` is a symlink, unlink it, then move the global
```

Comment:
Could we use hardlinks instead of symlinks? That's that pnpm does and it seems to work great for a global cache but still have it linked to the local directory.

Reply:
I'm using symlinks as a means to check if the global and local directory have been cached. If I switch to hard links, I suppose I can simply check `pathExists()`, right?

Reply:
Where is the symlink currently being created? My thought is if you create a hardlink instead, you will never need to check `pathExists()` just glob it every time.